### DISCUSSION TITLE
color-picker透明度

### DISCUSSION THREAD

**Discussion by Bland-he** [ant-design/ant-design#47303] [Ideas]
可以增加一个api来设置color-picker的初始透明度，新增另一个api在选择透明度后再选择另一个颜色的时候不要重置透明的为100%，保留上次已选择的透明度

**Comment by zombieJ** [comment]
听起来直接给个默认带透明度的颜色就行了？~

**Reply by Bland-he** [reply]
现在的颜色默认透明度是100%，可以在外部设置一个初始值为15%，这样在选择颜色的时候就是为15%不是100%。在选择透明度之后保留之前的透明度15%。const Demo = () => <ColorPicker defaultValue="#1677ff" defaultAlpha="15%" extendsAlpha=true />;

**Comment by MadCcc** [comment]
透明度是颜色的一部分，直接拖动面板是不会修改透明度的，你是点到预设颜色没有透明度的了吧，建议自己给一套有透明度的预设颜色。
